src/components/Header.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  addTodo: (todo: string) => void,
+  updateTodo: (uTodo: Todo) => void,
+};
+
+export const Header: React.FC<Props> = ({ todos, addTodo, updateTodo }) => {
+  const isSomeActive = todos.some(todo => !todo.completed);
+
+  const [value, setValue] = useState('');
+
+  const handleFormSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    addTodo(value);
+    setValue('');
+  };
+
+  const handleCompleteAll = (e: React.MouseEvent) => {
+    e.preventDefault();
```

Comment:
You don’t need to call 'preventDefault' on a click event unless it’s a submit button or a link. It's redundant in this case.